### DISCUSSION TITLE
Weekly meeting notes

### DISCUSSION THREAD

**Discussion by spwoodcock** [hotosm/field-tm#1886] [General]
- We hold weekly meetings on FMTM development at 9:30 UTC every Wednesday.
- Join the [HOTOSM Slack](https://slack.hotosm.org) and send us a message to get involved.

This discussion records the minutes from our sessions.

**Comment by spwoodcock** [comment]
# 2024-11-13

## Details

- Time 3:15 pm NPT
- Attendees:
  - HOT: Ramya, Sam
  - NAXA: Manjita Pandey, Sujan Adhikari, Nishit Suwal, Anuj Gupta
  - External: none

## Agenda

1. **Manjita**: [Visualising user uploaded photos on submission details page](https://github.com/hotosm/fmtm/issues/1706) - can we prioritise visualising all images uploaded for tokha field mapping project? They are expecting to use the FMTM for data validation too and this is a vital part of validation.

2. **Manjita**: Removing the categories from FMTM.

## Notes

- The functionality for external storage of submission photos is available in the latest version of ODK Central, which was upgraded recently.
- Need to investigate how the external storage functionality is implemented in ODK Central and to set it up if it is important for Tokha.

**Comment by manjitapandey** [comment]
**2025- 01-01**
--

Time 3:15 pm NPT
Attendees:
HOT: Ramya,
NAXA: Manjita Pandey, Sujan Adhikari, Nishit Suwal, Anuj Gupta

**Agenda**

- Plan to push changes to staging by this week
- How is the priority for assigning tasks to mapper?(project level assigning or task level assigning)


**Notes**

- The new changes will be pushed to staging by this week including pulse effect and reverting automatic Geolocation. 
- Storing bad submissions in a separate table instead of directly getting them from osm to increase performance. #1986 
- Next meeting will be scheduled on monday to discuss around user management including User Roles and permission, type of projects, assigning tasks to users and other login options) 
- Enhancement done in user navigation from user's location to selected feature by limiting API calls every 10 seconds instead of every milliseconds. #1888
